src/Components/Programs/Timeline.js

```diff
@@ -79,7 +79,7 @@ function Timeline() {
                             {
                                 events.map((item,index)=>(
                                     <View style={{marginBottom:32,flexDirection:'column'}} key={index}>
-                                        <Line style={{left:(item.date[0][1]-1)*40, width:30*12*50, borderBottomColor:item.color}}></Line>
+                                        <Line style={{left:(item.date[2][1]-1)*40, width:30*12*50, borderBottomColor:item.color}}></Line>
```

Comment:
Why is 'date[2][1]' taken here ?

Reply:
because the date[2][1] value is nearly same for both the events GSoC and rail girls but date[0][1] value is not same for both